### PR TITLE
Invoice preview

### PR DESCRIPTION
**Pull request by specialunderwear**
This PR adds a preview function, that allows staff members to view html versions of the invoices. 
It is almost undoable to change the template without being able to view the invoice as html.
The stored document is still not accesible via internet.

### CODE REVIEW

**Comment by specialunderwear** [comment]
The build fails because it picks up some weird django version.

**Comment by sasha0** [comment]
Feel free to adjust configuration if it will fix the build.

**Comment by specialunderwear** [comment]
No idea why tox still install django 2.2a1 when I specified no pre releases should be used.
pip_pre = false should block any prereleases. Also I don;t understand why tox does not install the specified django, in deps it is clearly specified the test env should contain django>=2.0,<2.1 but it just runs setup.py without properly settings up the test environment, which causes a download of a crappy django version.

**Comment by mj026** [comment]
@specialunderwear https://github.com/specialunderwear/django-oscar-invoices/pull/1

File: oscar_invoices/views.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+def invoice(request, pk):
+    iv = Invoice.objects.get(pk=pk)
+    ic = InvoiceCreator()
+    rendered_template = ic.render_document(
+        iv,
+        legal_entity=iv.legal_entity,
+        order=iv.order,
+        use_path=False,
+    )
+    return HttpResponse(rendered_template)
```
Comment (sasha0) [inline oscar_invoices/views.py:20]: Any particular reasons not to make it CBV?

**Pull request merged**